src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -597,7 +953,20 @@ static void Main()
             var comp = CreateCompilationWithMscorlib(source, parseOptions: TestOptions.ExperimentalParseOptions, options: TestOptions.DebugExe);
             comp.VerifyDiagnostics();
             CompileAndVerify(comp, expectedOutput: "0");
-            // PROTOTYPE(default) Verify semantic model. What is the type of default?
+
+            var tree = comp.SyntaxTrees.First();
+            var model = comp.GetSemanticModel(tree);
+            var nodes = tree.GetCompilationUnitRoot().DescendantNodes();
+
+            var def = nodes.OfType<DefaultLiteralSyntax>().Single();
+            Assert.Null(model.GetTypeInfo(def).Type);
+            Assert.Null(model.GetSymbolInfo(def).Symbol);
+            Assert.Null(model.GetDeclaredSymbol(def));
+
+            var conversion = nodes.OfType<CastExpressionSyntax>().Single();
+            var conversionTypeInfo = model.GetTypeInfo(conversion);
+            Assert.Equal("System.Int16", conversionTypeInfo.Type.ToTestDisplayString());
+            Assert.Equal("System.Int32", conversionTypeInfo.ConvertedType.ToTestDisplayString());
```

Comment:
Oops, sorry, I misunderstood. I tested access, but not array construction. I'll add.